### PR TITLE
Fix fans and ender air tips typo

### LINKED ISSUE #4536: [E6E]The custom tips ender_air_ghasts.json and astral_cake.json have the same text.
### Modpack Version

1.1.1

### Describe your issue.

They have the same text "Astral Sorcery can allow for easy Kekimurus automation."

### Crash Report

_No response_

### Latest Log

_No response_

### Have you modified the modpack?

No

### User Modifications

_No response_

### Did the issue happen in singleplayer or on a server?

Singleplayer

### PR DESCRIPTION
**Pull request by dodieboy**
fix #4536

### CODE REVIEW

File: kubejs/assets/enigmatica/tips/fans.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
     "tip": {
-        "text": "Dispensing Ender Air at Ghasts in the Overworld gets them all teary-eyed! -MuteTiefling"
+        "text": "A single fan is enough to turn a pair of crusher wheels at the default speed."
```
Comment (Darkere) [inline kubejs/assets/enigmatica/tips/fans.json:3]: Crushing Wheels?

Comment (dodieboy) [reply kubejs/assets/enigmatica/tips/fans.json:3]: yup, crushing Wheels from create mod.

Comment (Darkere) [reply kubejs/assets/enigmatica/tips/fans.json:3]: Yes... Then what are crusher wheels? :D

Comment (dodieboy) [reply kubejs/assets/enigmatica/tips/fans.json]: my bad, typo

**Review by NillerMedDild — APPROVED** [review]

**Comment by NillerMedDild** [comment]
Thanks for the fix!

**Pull request merged**